src/odemis/acq/stream/_base.py

```diff
@@ -191,6 +193,8 @@ def __init__(self, name, detector, dataflow, emitter, focuser=None, opm=None,
         self._det_vas = self._duplicateVAs(detector, "det", detvas or set())
         self._emt_vas = self._duplicateVAs(emitter, "emt", emtvas or set())
 
+        self._init_axis_vas(axis_map)
```

Comment:
It'd fit better the code if it could look like:
`self._axis_vas = self._duplicateAxes(...).`

Also, IMHO, it's nice if the VAs are available directly as attributes of the stream. So, for each axis, also provide the VA as an attribute `.axisNameOfTheAxis`.